.azure/azure-pipelines.ci.yml

```diff
@@ -39,12 +41,79 @@ stages:
         arguments: --configuration $(buildConfiguration)
     - publish: $(System.DefaultWorkingDirectory)/nupkg
       artifact: ToolNuPkg
+- stage: tests
+  displayName: Tests
+  dependsOn:
+  - build
+  jobs:
+  - job: build_example_linux
+    displayName: Build Example Linux
+    pool:
+      vmImage: ubuntu-latest
+    steps:
+    - task: UseDotNet@2
+      displayName: 'Use .NET Core sdk'
+      inputs:
+        packageType: sdk
+        version: 3.1.302
+        installationPath: $(Agent.ToolsDirectory)/dotnet
+    - task: DownloadPipelineArtifact@2
+      displayName: Download Artifacts
+      inputs:
+        artifact: ToolNuPkg
+        path: $(System.DefaultWorkingDirectory)/nupkg
     - script: |
+        ls $(System.DefaultWorkingDirectory)/nupkg
         dotnet tool install --global --add-source $(System.DefaultWorkingDirectory)/nupkg Microsoft.Logging.CLOG
         mkdir build
         clog --installDirectory build/clog
         cd build
         export CLOG_DEVELOPMENT_MODE=1
         cmake ../examples
-        make
+        cmake --build .
+      displayName: Test Sample Build
+  - job: build_example_windows
+    displayName: Build Example Windows
+    pool:
+      vmImage: windows-latest
+    steps:
+    - task: DownloadPipelineArtifact@2
+      displayName: Download Artifacts
+      inputs:
+        artifact: ToolNuPkg
+        path: $(System.DefaultWorkingDirectory)/nupkg
+    - pwsh: |
+        dotnet tool install --global --add-source $(System.DefaultWorkingDirectory)/nupkg Microsoft.Logging.CLOG
+        mkdir build
+        clog --installDirectory build/clog
+        cd build
+        $env:CLOG_DEVELOPMENT_MODE = 1
+        cmake ../examples
+        cmake --build .
+      displayName: Test Sample Build
+  - job: build_example_macos
+    displayName: Build Example macOS
+    pool:
+      vmImage: macOS-10.15

```

Comment:
Looks like macOS still isn't working. Let's just remove it from the CI for now, and open an issue for onboarding support.

Reply:
Its just 10.15 systems that are down. 10.14 works, so going with that

Reply:
Sounds good.